Modules/xxlimited_35.c

```diff
@@ -247,43 +245,24 @@ xx_modexec(PyObject *m)
     /* Add some symbolic constants to the module */
     if (ErrorObject == NULL) {
         ErrorObject = PyErr_NewException("xxlimited_35.error", NULL, NULL);
-        if (ErrorObject == NULL) {
-            return -1;
-        }
     }
-    Py_INCREF(ErrorObject);
-    if (PyModule_AddObject(m, "error", ErrorObject) < 0) {
-        Py_DECREF(ErrorObject);
+    if (PyModule_AddObjectRef(m, "error", ErrorObject) < 0) {
         return -1;
     }
 
     /* Add Xxo */
     Xxo_Type = PyType_FromSpec(&Xxo_Type_spec);
-    if (Xxo_Type == NULL) {
-        return -1;
-    }
-    if (PyModule_AddObject(m, "Xxo", Xxo_Type) < 0) {
-        Py_DECREF(Xxo_Type);
+    if (PyModule_Add(m, "Xxo", Xxo_Type) < 0) {
```

Comment:
It's not part of the limited C API 3.5: GHA compiler warning: implicit declaration of function ‘PyModule_Add’; did you mean ‘PyModule_New’? [-Werror=implicit-function-declaration]